workbench/shared.php

```diff
@@ -772,7 +772,8 @@ function crypto_serialize($data) {
 }
 
 function crypto_unserialize($data) {
-    return unserialize(base64_decode(rc4($data, WorkbenchConfig::get()->value("rc4Secret"), false)));
+    $results = unserialize(base64_decode(rc4($data, WorkbenchConfig::get()->value("rc4Secret"), false)));
```

Comment:
@rvwatch - Maybe you can remove this change from this PR and then commit it as-is. That way we have one single commit that links up Redis locally that can be merged into master first.

Reply:
This should be updated now.